scipy/optimize/_highs/cython/src/_highs_constants.pyx

```diff
@@ -56,7 +56,6 @@ from .SimplexConst cimport (
     SIMPLEX_EDGE_WEIGHT_STRATEGY_DANTZIG,
     SIMPLEX_EDGE_WEIGHT_STRATEGY_DEVEX,
     SIMPLEX_EDGE_WEIGHT_STRATEGY_STEEPEST_EDGE,
-    SIMPLEX_EDGE_WEIGHT_STRATEGY_STEEPEST_EDGE_UNIT_INITIAL,
```

Comment:
This was used in commented-out code, so could have been left. But it doesn't really matter either way, so let's go with the change proposed here.